core/lib/types/src/api/en.rs

```diff
@@ -46,3 +46,10 @@ pub struct SyncBlock {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ConsensusGenesis(pub serde_json::Value);
+
+/// Status of L1 simulated by the main node.
+/// Used for testing L1 batch signing by consensus attesters.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SimulatedL1Status {
+    pub next_batch_to_commit: L1BatchNumber,
```

Comment:
We do indeed try to simulate L1, but I would still use a more straightforward/descriptive namings for this API. We simply return the first batch that doesn't have quorum yet. I would model the API correspondingly - why load it with this additional context that in the future this will be L1? IMO this feature makes sense even without L1 - we want to collect signatures without any gaps, so we return the current state there.

@pompon0 what do you think?

Reply:
I also want to note that I think it will take months to get this on L1 (due to audits and protcol upgrade functionality), so, while temporary, it's not like it's only there for some days/weeks. We may even advertise this publicly before this is on L1.

Reply:
I would agree that `SimulatedL1Status` is muddying the waters here - I can't even imagine what else people might think they could stuff in there about L1, chain height maybe? `L1BatchStatus`, perhaps?

Reply:
I'm not returning just a number because then the call won't be extensible. I want to be prepared to return more info here if needed. The simulated L1 status is the least common denominator of what I want to put here. I can rename it to sth like GlobalAttestationStatus and next_batch_to_attest if you want. This API is supposed to provide an abstraction - i.e. not leak how stuff is represented internally. I also want to make it really obvious that this is just a placeholder and that by itself it doesn't provide value.

Reply:
> I want to be prepared to return more info here if needed.

What additional info do you have in mind?

>  I can rename it to sth like GlobalAttestationStatus and next_batch_to_attest if you want

I would definitely prefer it - `LastAttestestedBatch` would be even simpler.

> I also want to make it really obvious that this is just a placeholder and that by itself it doesn't provide value.

Thing is, the feature that we are building currently is being able to collect signatures on the leader node. The next step will be L1, but the feature has value without L1. This feature needs an API - so let's build this API - in this sense it does provide value.

Overall the naming itself is not very important - but I wanted to raise it to underline that for now we don't do anything with L1, so let's not introduce such concepts in code

Reply:
`chain height` and `L1BatchStatus` are names neither related to attestation nor the fact that this rpc does NOT provide information related to the real chain state.

Reply:
yeah, that's intentional. We may add for example the info that you considered, i.e. how many batches in parallel we want to collect signatures for.

Reply:
> LastAttestestedBatch would be even simpler.

This is not extensional name. And note that the EN rpc namespace already is a collection of random stuff. If we end up in a situation that we need more than 1 rpc, I don't want to have a bunch of them mixed among other unrelated rpcs in the same namespace. If you want this rpc to be so very specific (with a name lastAttestestedBatch or similar) then I vote to put it in a separate RPC namespace, which will be just for attestation-related rpcs.

Reply:
The examples of extension you mention are still related to batch signing, which is why I proposed `L1BatchStatus`. 

All I'm trying to point out that `SimulatedL1Status` might compel somebody else who has some L1 related thing to add it to this endpoint, even when it's unrelated to what we are doing now, leading to shared ownership of what we say is a temporary API. For example the last finalised block height on L1. If that is intentional, then I rest my case.

Reply:
renamed to AttestationStatus. Is that ok?